src/main/java/QuizQuestionPresenter.java

```diff
@@ -0,0 +1,74 @@
+import io.reactivex.Observable;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.schedulers.Schedulers;
+import json.NewQuestion;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class QuizQuestionPresenter {
+
+    private final QuizQuestionFrame view;
+    private final GetQuizQuestion model;
+    private Disposable disposable;
+    private int placementOfCorrectAnswer;
+
+    public QuizQuestionPresenter(QuizQuestionFrame view, GetQuizQuestion model)
+    {
+        this.view = view;
+        this.model = model;
+    }
+
+    public void nextQuestion()
+    {
+        Observable<NewQuestion> question = model.getQuizQuestion();
+
+        disposable = question
+                .subscribeOn(Schedulers.io())
+                .observeOn(Schedulers.newThread())
+                .subscribe(this::onNext, this::onError);
+    }
+
+    public void cancel() {
+        if (disposable != null) {
+            disposable.dispose();
+        }
+    }
+
+    private void onNext(NewQuestion question) {
+        view.setQuestion(question.getQuestion1());
+
+        String[] answers = question.getIncorrectAnswers1();
+        String correct = question.getCorrectAnswer1();
+
+        ArrayList<String> abcOrder = new ArrayList<>();
+        abcOrder.add(answers[0]);
+        abcOrder.add(answers[1]);
+        abcOrder.add(answers[2]);
+        abcOrder.add(correct);
+
+        Collections.sort(abcOrder);
```

Comment:
I think you want to `shuffle()` the answers, not `sort()` them but perhaps it doesn't matter.

Reply:
I though ABC order would be neater (especially for questions that will have numbers as answer choices)